js/src/frontend/smoosh/smoosh.h

```diff
@@ -3,9 +3,9 @@
 #include <cstdlib>
 #include <new>
 
-template <typename T>
+template<typename T>
 struct CVec {
-  T* data;
+  T *data;
```

Comment:
It looks like this generated file wasn't clang-formatted on commit. I don't know if it should be or not, but it probably shouldn't flip-flop.

Reply:
I'll add this to https://searchfox.org/mozilla-central/source/.clang-format-ignore